derive_builder/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+- Requires Rust 1.40.0 or newer (was 1.37.0) #169
```

Comment:
Can you update the badge in `README.md` as part of this change, and set both crate versions to 0.10.0-alpha just to avoid any mishaps?

Reply:
Badge is already updated. Root README.md is a symlink to derive_builder/README.md